templates/zerver/help/changing-the-default-language.md

```diff
@@ -0,0 +1,27 @@
+# Changing the Default Language
+
+Zulip is written primarily in English, but it includes feature support for a variety of languages. Changing the default language in Zulip will translate many of the settings and menu options into the language that you select. Because some languages are only partially supported,  you may still see English words interspersed throughout Zulip.
+
+## Summary:
+
+-  Click the gear icon (![Gear Icon](/static/images/help/cog.png)) in the upper right-hand corner of Zulip and select **Settings**.
+
+-  Scroll down to the **Display Settings**.
+
+- Next to the **Default Language** heading, select the blue text: **[Change]**.
+
+    ![Display Settings Menu](/static/images/help/display-settings-menu.png)<br><br>
+
+- Finally, choose your desired language from the options in this popup menu:
+
+    ![Change Default Language Menu](/static/images/help/change-default-language-menu.png)<br><br>
+
+- The menu will close, and the language changes will take effect as soon as you **reload the page**.
+
+- And that's it! Now, you can enjoy Zulip in your desired language 😜.
+
+## Want to help out Zulip?
+
+You can help translate parts of Zulip by going to **[https://www.transifex.com/zulip/zulip/](https://www.transifex.com/zulip/zulip/)** and signing up for an account.<br><br>
+
+**[Go back to “Table of contents”](/help)**
```

Comment:
What do you think is the best way to provide context to each of the screenshots? A caption beforehand is what I was thinking, but I'm not sure if that's the best option.